extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -217,6 +217,10 @@ More details for adding INFOnline anonymous support can be found at [www.infonli
 
 Type attribute value: `iplabel`
 
+### iwa
+
+Type attribute value: `iwa`
+
```

Comment:
Can you add a quick description of what support iwa is?

Reply:
done